lib/storage/src/content_manager/consensus/persistent.rs

```diff
@@ -192,10 +204,17 @@ impl Persistent {
         Ok(state)
     }
 
+    fn load_json(path: PathBuf) -> Result<Self, StorageError> {
+        let file = File::open(&path)?;
+        let mut state: Self = serde_json::from_reader(&file)?;
+        state.path = path;
+        Ok(state)
+    }
+
     pub fn save(&self) -> Result<(), StorageError> {
         let result = AtomicFile::new(&self.path, AllowOverwrite).write(|file| {
             let writer = BufWriter::new(file);
-            serde_cbor::to_writer(writer, self)
+            serde_json::to_writer(writer, self)
```

Comment:
What about `serde_json::to_writer_pretty`? Much easier to work with, especially considering that `jq` "truncates" big integer values (e.g., node IDs). :/

```suggestion
            serde_json::to_writer_pretty(writer, self)
```

Reply:
> especially considering that jq "truncates" big integer values (e.g., node IDs).

This part should be fixed in latest versions, we are no longer generating that big integers

Reply:
I am not sure about the overhead, though